articles/other/other-faq-g12.md

```diff
@@ -0,0 +1,148 @@
+
+### What is VMware Licence Service?
+
+The VMware Licence Service (VLS) is an innovative managed service designed with VMware to make it easier to modernise VMware environments as organisations migrate to the cloud. VLS enables the exchange of traditional VMware SnS with a subscription based managed service provided by UKCloud as part of the journey to fully adopting a multi-cloud strategy. It offers a flexible monthly pay as you go licence service as an alternative to the expensive inflexible upfront costs for VMware support.
+
+### What is Neustar UltraDNS from UKCloud?
+
+Neustar UltraDNS is an enterprise grade, cloud-based authoritative DNS service that securely delivers fast and accurate query responses to websites and other vital online assets. Whether you're looking for a primary or secondary DNS solution, Neustar UltraDNS offers customizable packages to fit any organisation's DNS needs. The UltraDNS Firewall option is a cost-effective enterprise grade, cloud-based recursive DNS service that delivers fast and reliable access to vital online applications with built-in security and threat intelligence. Although previously offered as a service option, this is now a separate service with its own Service Definition.
+
+### What is Neustar DDoS Protection from UKCloud?
+
+UKCloud's internet-connected Assured OFFICIAL platform already has Neustar DDoS mitigation baked in, however this mitigation is aimed at protecting the entire platform and is based on UKCloud's aggregate traffic profile. Neustar's UltraDDos Protect and UltraWAF are tailored to your own organisation's traffic patterns, providing you with powerful analytics, top-tier personalised DDoS mitigation and layer 7 protection that enables you to suppress threats before they become attacks. Although previously offered as a service option, this is now a separate service with its own Service Definition.
+
+## Changes to existing services
+
+For information about any of the updated services listed below, see [*UKCloud Service Definitions and Pricing Guide*](https://docs.ukcloud.com/articles/other/other-ref-service-definitions.html).
+
+### What's new in the UKCloud for OpenShift service?
```

Comment:
this service now offers customers on OCPv4 or higher, the option for...

Reply:
Done